__mocks__/request-promise.js

```diff
@@ -0,0 +1,47 @@
+import {getConfig} from '../src/__mocks__/helpers';
+
+
+function request(url) {
+  // TODO get rid of those globals
+  switch (global.MOCK_REQUEST_SCENARIO) {
+    case 'broker-latest-pact':
```

Comment:
I mock the request-promise library. It should return different files in different cases. Can you exemplify?